### DISCUSSION TITLE
Cannot use pythonnet to load dlls in a python module that itself is being loaded with pythonnet in C#

### DISCUSSION THREAD

**Discussion by jbrownkramer** [pythonnet/pythonnet#2065] [Q&A] [answered]
### Environment

-   Pythonnet version: 3.0.1 nuget package.  pythonnet-3.0.1 on the python side.
-   Python version: 3.8
-   Operating System: Windows
-   .NET Runtime: 5.0 for the C# and the target framework for the DLL I am trying to load in python in .net 4.7.2

### Details

-   Describe what you were trying to get done.

    I have script.py which has a clr.AddReference("foo") in it and then import foo.  It works when running python script.py.  However, if I do Py.Import("script"), in a C# script, I get the error "No module named foo".  I have checked that the directory containing foo.dll is in sys.path even when importing through C#
    
    Is there some fundamental reason I can't do this, or is it more likely to be some pathing issue?

**Comment by filmor** [comment, marked as answer]
The only fundamental problem I can see is the mix of runtimes that you list here. Can you please create a reproducible example for this case?

**Reply by ghost** [reply]
Is foo.dll is part of Debug folder of your C# script? If not try placing it in Debug folder and check if that helps.
